packages/nuxt/README.md

```diff
@@ -0,0 +1,11 @@
+# @leanjs/nuxt
+
+## Installation
+
+`yarn add @leanjs/next @leanjs/react`
+
+## Limitations
+
+Due to the nature of nuxt, we currently cannot support microfrontends that manage multiple routes in nuxt.
```

Comment:
Maybe we could say why it's not supported and add a link to a feature request to Nuxt?

Reply:
Good shout - I'm gonna merge this now and raise a request in another PR :+1: